hadoop-ozone/dev-support/checks/install/_install.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Tricky name, as this script is responsible both for install and executing the scripts.

I am wondering if it would be easier to do a laze download at the beginning of each of the scripts. For example at the beginning of the findbugs.sh we can check if spotbugs exists and download if not.

Reply:
> Tricky name, as this script is responsible both for install and executing the scripts.

This is just a wrapper to invoke a specific install script.  It does not run the check.  The only reason for its existence: the workflow file can blindly invoke it without checking for the existence of the specific install script (I'm not even sure it would be possible).